### DISCUSSION TITLE
Using Laragon 6 Full with different ports

### DISCUSSION THREAD

**Discussion by jrapusas** [leokhoa/laragon#438] [Q&A]
Hello everyone!

Laragon is great using the default ports for Apache and MySQL, but I need to have Laragon work on different ports.

I have XAMPP running on Port 80 and 3306 for Apache and XAMPP.

I install Laragon 6 Full. Upon install I change the Services and Ports to the following:
Apache: 8089
MySQL: 3309

When I Start All, the Apache does not work, and the MySQL is stuck at initializing.

What else needs to be done so that it will work on my desired ports?

Thanks in advance.

**Comment by Pen-y-Fan** [comment]
My first thought is something else is running on port 3309, to check this Laragon menu > Tools > Netstat

![image](https://user-images.githubusercontent.com/40126936/224639192-24e68839-0008-45c7-89a9-c604754993e5.png)

Wait for Notepadd++ to start and check nothing else is running on ports 8089 and 3309, if there is try another port e.g. 8888/33061.

The next thing I would check is permissions, the installer runs under elevated permissions. I always set the standard user account to have full permission. See my blog [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/). 

Have you restarted after the installation? That would be the next step, as this would clear any blocking or conflicting ports.

I hope this helps.
